lib/geolocation/simulator.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
```

Comment:
add similar header of other files.

Reply:
Can I simply assign copyright to Samsung one-sidedly? I mean, I don't really care about this code so I'm happy to give it to you, but it feels odd that I can claim Samsung owns something without Samsung agreeing to that?

Reply:
good catch, in this case I can create the file 1st and then you will rebase on it (and squash your commits)
does this sound good ?

Reply:
Haha fine by me, not sure if that counts legally, but then again, I guess it's mostly pedantry anyway :)